.github/workflows/ci.yml

```diff
@@ -26,19 +23,51 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: "npm"
       - name: Continuous Integration
-        run: npm ci
-      - name: Install Dependencies
+        run: npm ci 🔄
+      - name: Install Dependencies 🔧
         run: npm install
```

Comment:
We do not want to run `install` and `ci` here, what we want is probably only a `npm ci`.
https://docs.npmjs.com/cli/v8/commands/npm-ci